### ISSUE TITLE
Settings added by Error Lens in settings.json is throwing error

### ISSUE THREAD

**Issue by anandinasu** [phindle/error-lens#42]
![image_109](https://user-images.githubusercontent.com/12847193/132557201-e5552e10-187e-4407-938d-3130555920a4.png)

**Comment by kod4x** [comment]
hi, yes, i noted that too, maybe putting the urls in a json made the errors for the "//", i uninstalled the extension to test it and is the only one making the errors in the file.

**Comment by anandinasu** [comment]
Yes, that is true. The error is coming because of Error Lens settings and it is because the websites are not commented. I don't know why it is not commented. Also, they could use block comments for all the deprecated settings.
